src/compiler/driver.cc

```diff
@@ -78,16 +76,41 @@ struct VisitorPostCompilation : public hilti::visitor::PreOrder<void, VisitorPos
 
 Driver::Driver(const char* argv0, hilti::rt::filesystem::path plugin_path, int zeek_version)
     : spicy::Driver("<Spicy Plugin for Zeek>", argv0) {
+    // TODO: The following initLocatio() is just work-around. Remove once
+    // spicy::Driver() allows to skip argv0 altogether to achieve the same
+    // effect.
+    hilti::configuration().initLocation(false);
     spicy::Configuration::extendHiltiConfiguration();
+    auto options = hiltiOptions();
 
-    plugin_path = hilti::rt::filesystem::canonical(plugin_path);
+    // TODO: Do we need this? There's already `HILTI_PATH` and `SPICY_PATH`.
+    if ( auto path = getenv("ZEEK_SPICY_PATH") ) {
+        for ( const auto& dir : hilti::rt::split(path, ":") ) {
+            if ( dir.size() )
+                options.library_paths.emplace_back(dir);
+        }
+    }
 
-    auto options = hiltiOptions();
+    // TODO: Can remove this once HILTI provides an environment variable to
+    // speicy include directories.
```

Comment:
Is this something to do before merging?
```suggestion
    // TODO: Can remove this once HILTI provides an environment variable to
    // specify include directories.
```